PRODUCTIONISATION.md

```diff
@@ -64,11 +64,10 @@ Ensuring that your project adheres to the following RFCs is a good first step as
 
 * 5.1 Continuous Delivery pipeline configured to:
   * [ ] 5.1.1 Automatically test pull requests
-  * [ ] 5.1.2 `master` branch to run linting and tests
+  * [ ] 5.1.2 `master` branch to run tests
```

Comment:
flagging, as there's a discussion around linting going on. This may need restoring depending on the outcome

Reply:
Resolved in https://github.com/madetech/productionisation/pull/51/commits/313b3e95bdbbb42b359338311e99e7b73b4e84cb.